capstone-app/src/app/modules/history/wiki-media/wiki-media.component.ts

```diff
@@ -1,9 +1,46 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { WikiSearchResult } from '../WikiSearchTemplate';
+import { WikiResultsService } from '../wiki-results.service';
+import { mergeMap, switchMap, map, takeUntil } from 'rxjs/operators';
+import { Subject } from 'rxjs';
 
 @Component({
-  selector: 'app-wiki-media',
+  selector: 'app-wiki-media-history-section',
   templateUrl: './wiki-media.component.html',
   styleUrls: ['./wiki-media.component.scss']
 })
-export class WikiMediaComponent{
+export class WikiMediaComponent implements OnInit, OnDestroy {
+  wikiResult: WikiSearchResult;
+  destroy$ = new Subject<void>();
+  loading = true;
+  body: string;
+  constructor(private wikiService: WikiResultsService) { }
+
+  ngOnInit(): void {
+    this.getResults();
+  }
+
+  ngOnDestroy(): void {
+    this.destroy$.next();
+  }
+
+  getResults() {
+    this.wikiService.search('Orlando, Florida').pipe(takeUntil(this.destroy$)).subscribe((result: WikiSearchResult) => {
```

Comment:
Also, this doesn't need to be added in this pull request (perhaps in a subsequent pr), but some error handling should be added here if the request fails. It should probably display a message if the request fails, such as "Failed to fetch location history" or something.

If you add error handling, add a test for it as well.

If you don't add error handling right now, add a TODO:

```
// TODO(sarahhud): Add error handling for wikipedia request.

Reply:
Also, add a TODO to replace the 'Orlando, Florida' with a real string, once the connection with the search bar / google maps is established.

Reply:
Yes, I was planning on handling errors in another pr. I will add a to-do.